### ISSUE TITLE
Export to PDF results in empty document

### ISSUE THREAD

**Issue by erno1978** [giuspen/cherrytree#1774]
**Version, Operative system**
0.99.40, Windows

**Describe the bug**
Export to PDF doesn't work if "selected node with subnodes" or "full tree" is ticked. The result is an empty PDF. If I try the same on the (0.99) version of the Linux variant of CherryTree it works perfectly.

Note: plain export of one node works as well perfectly. It's something with subnodes...

**Comment by erno1978** [comment]
Hi Giuseppe,

Sure! I included the Test CherryTree document. More info you might need:

   - I tested it on Windows 10 (version 0.99.40 of CherryTree)
   - The document is newly created and really easy, I'm trying to export
   node "b" with the 2 subnodes included. That doesn't work like I stated in
   the request.
   - Also the cherrytree.log is included

If there's more I can do to clarify, let me know. I love the product, by
the way!

Cheers!
Ernst



Op vr 27 aug. 2021 om 21:39 schreef Giuseppe Penone <
***@***.***>:

**Comment by giuspen** [comment]
@erno1978 I cannot find any attachment, in order to attach to the github post you have to access github I think, you cannot answer via email

**Comment by erno1978** [comment]
[cherrytree.log](https://github.com/giuspen/cherrytree/files/7072033/cherrytree.log)

**Comment by erno1978** [comment]
@giuspen Sorry, I don't use github much, but I just updated my first reply with the attachment.

**Comment by erno1978** [comment]
> Unfortunately it's working for me, I get a proper pdf with the three nodes. What language is your OS? Maybe a particular pdf generator installed on windows? If you backup the config.cfg (just rename it when cherrytree is not running) and try with the default settings is it happening again?

It's crazy! I did several tests just now. I am running the latest/patched Windows 10 Home. If I'm doing the export on the issue.zip file included it's didn't work (again), when I removed the config.cfg file from my AppData it did actually work! And to make things even stranger: when I replaced the config.cfg again with my backupped version (yeah, the one that didn't work), it's also working for the test.ctb.

My problem isn't fixed by the way, because when I try my notes (that's like 50 pages, full of screenshots and bullet lists) the export function makes my CherryTree crash (it takes 10+% CPU power). I have a powerful laptop, lots of memory, no strange other PDF export software or whatsoever.

But I guess this new issue is not really related to this issue, so I'm good with you closing this issue. I will think/test some more the next few days/weeks and if I get a good test scenario I will open another issue?

Thanks for your  time/support!
